src/api/__tests__/Versions-test.js

```diff
@@ -43,14 +43,12 @@ describe('api/Versions', () => {
                 total_count: 2,
                 entries: [
                     {
-                        ...deleteVersion,
-                        action: 'delete',
-                        version_number: 1
```

Comment:
Since version_number now gets returned w/ the API response for versions, that field should probably remain in the tests

Reply:
👍 doesn't change the test, but perhaps it would be good to extract the stubs into some constants file so we don't have to keep reimplementing them. Will create a ticket for that